src/app/calendar-app/runbox-calendar-event.ts

```diff
@@ -258,11 +561,12 @@ export class RunboxCalendarEvent implements CalendarEvent {
             id:       this.id,
             calendar: this.calendar,
             jcal:     this.ical.toJSON(),
+            ical:     this.toIcal(),
```

Comment:
Wrong place for this comment (Github won't let me add one to unchanged lines), but we should probably remove the `// returns jCal` comment from this since it hasn't been correct for a while, from the looks of it :) Actually, I wonder: do we really need both `jcal` and `ical` in there?

Reply:
Nope, and since I updated the cache version anyway, might as well remove it

Reply:
.. in fact we're not using it at all, the saveCache does 

```({'id': event['id'], 'ical': event['event'].component.parent.toString() } )```